os/board/rtl8720e/Kconfig

```diff
@@ -19,6 +19,13 @@ config FLASH_SIZE
         A size of flash (in bytes).
         This is fixed value, so user doesn't need to change it.
 
+config FLASH_STATUS_BITS
+	hex "Flash Status Register Bits"
+	default 0x2c
+	---help---
+		Setting this value according to the IC package's Flash Model.
+		To protect the lower 512KB of the flash memory.
```

Comment:
Is this configurable? If not, let's remove `"Flash Status Register Bits"`. (Then, user can't change the value)

Reply:
It was NOT configurable. I will remove the "Flash Status Register Bits".
Thank You.